app/celery/celery.py

```diff
@@ -1,11 +1,130 @@
+from datetime import timedelta
+
 from celery import Celery
+from celery.schedules import crontab
+from kombu import Queue, Exchange
 
+from app.celery import QueueNames
+
+
+class CeleryConfig:
+    def __init__(self, config):
+        self.broker_transport_options['queue_name_prefix'] = config['NOTIFICATION_QUEUE_PREFIX']
+        self.broker_url = config.get('BROKER_URL', 'sqs://')
+
+    broker_transport_options = {
+        'region': 'sqs.eu-west-1',
+        'polling_interval': 1,  # 1 second
+        'visibility_timeout': 310,
```

Comment:
long polling means that SQS will keep the request open until it gets a message - but after it returns, celery still needs to know how long to wait until asking for the next one - this is the `polling_interval` (i think)